roles/openshift_common/meta/main.yml

```diff
@@ -13,4 +13,4 @@ galaxy_info:
   - cloud
 dependencies:
 - role: openshift_facts
-- role: openshift_repos
+- role: openshift_docker
```

Comment:
The conditional makes sense to me here, we want to avoid installing something where it isn't needed